### PR TITLE
“Schema generator“ > improve “Getting started”

### PR DESCRIPTION
**Pull request by alexislefebvre**
See #442

### CODE REVIEW

File: schema-generator/getting-started.md [hunk 1]
```diff
@@ -30,13 +35,7 @@ types:
             familyName: ~
             givenName: ~
             additionalName: ~
-            gender: ~
             address: ~
-            birthDate: ~
-            telephone: ~
-            email: ~
-            url: ~
-            jobTitle: ~
```
Comment (meyerbaptiste) [inline schema-generator/getting-started.md]: Maybe I missed something but why do you remove these properties?

Comment (alexislefebvre) [reply schema-generator/getting-started.md]: I think that we don't need 10 properties in order to understand this code sample. I make the generated code longer and it adds no value.

Comment (dunglas) [reply schema-generator/getting-started.md]: Agreed.

Comment (meyerbaptiste) [reply schema-generator/getting-started.md]: Makes sense.

**Pull request merged**